api_definition.go

```diff
@@ -699,6 +702,20 @@ func (a APIDefinitionLoader) compileTrackedEndpointPathspathSpec(paths []apidef.
 	return urlSpec
 }
 
+func (a APIDefinitionLoader) compileValidateJSONPathspathSpec(paths []apidef.ValidatePathMeta, stat URLStatus) []URLSpec {
+	urlSpec := []URLSpec{}
```

Comment:
To avoid slice re-allocation I think we could use `urlSpec := make([]URLSpec, len(paths))` and then do `for index, pathVal .. urlSpec[index] = newSpec`

Reply:
I like this idea - but I think for consistency, we keep same as rest of `api_definition.go`. Will create separate GH refactor issue for avoiding slice re-allocation.

Reply:
why introduce new code which will be subject of refactoring later? I mean we can refactor existing code as separated task for sure, but new changes - why not to do them better at the very beginning (if we believe they are better)